code/controllers/subsystem/starmap.dm

```diff
@@ -341,7 +341,7 @@ SUBSYSTEM_DEF(starmap)
 	ftl_drive.status_update(message)
 
 /datum/controller/subsystem/starmap/proc/ftl_sound(var/sound) //simple proc to play a sound to the crew aboard the ship, also since I want to use minor_announce for the FTL notice but that doesn't support sound
-	for(var/area/shuttle/ftl/F in world)
+	for(var/area/shuttle/ftl/F in world - /area/shuttle/ftl/cargo/mining)
```

Comment:
check if the shuttle is docked or not before excluding it, or check in the area if it's currently on the same z level as the ship.

Reply:
do the second actualy

Reply:
Done but my method is probably equally terrible.